requirements-dev.txt

```diff
@@ -4,3 +4,5 @@
 # Install with `pip3 install -r requirements-dev.txt`
 
 flake8==3.7.8
+filecheck==0.0.17
+lit==0.11.0.post1
```

Comment:
I'm not familiar with Python ports of filecheck and lit... So lit needs a wrapper script but filecheck can run as is like a normal executable?